lib/neverblock.rb

```diff
@@ -21,7 +23,7 @@ def self.neverblocking?
   # mode based on the first parameter
   def self.neverblock(nb = true, &block)
     status = NB::Fiber.current[:neverblock]
-    NB::Fiber.current[:neverblock] = nb
+    NB::Fiber.current[:neverblock] = !!nb
```

Comment:
what do we pass besides true / false?

Reply:
As far as I can tell, should only ever be true/false - NB::Fiber.current[:neverblock] is only checked for truthiness in the NB.neverblocking? function